pkg/core/native/native_neo.go

```diff
@@ -444,8 +501,52 @@ func (n *NEO) SetGASPerBlock(ic *interop.Context, index uint32, gas *big.Int) (b
 	return true, n.putGASRecord(ic.DAO, index, gas)
 }
 
-// CalculateBonus calculates amount of gas generated for holding `value` NEO from start to end block.
-func (n *NEO) CalculateBonus(ic *interop.Context, value *big.Int, start, end uint32) (*big.Int, error) {
+func (n *NEO) checkCandidate(d dao.DAO, pub *keys.PublicKey, c *candidate) error {
```

Comment:
`dropCandidateIfZero`? I mean it's not a `check` really.

Reply:
Fixed.